README.md

```diff
@@ -25,4 +25,6 @@ BENEFICIARY=0x07b589f06bD0A5324c4E2376d66d2F4F25921DE1 \
   pnpm print-irrevocable-vesting-address
 ```
 
+_It is required that you also have a MNEMONIC environment variable set before runnning this script_
+
```

Comment:
Did this instead https://github.com/api3dao/irrevocable-vesting/commit/71bc7c0b10b24feb8adc774544bd47002069efd6